Software/tft/README.md

```diff
@@ -0,0 +1,17 @@
+# The Firmware Tool
+This is a simple tool to add header to raw firmware binary. The header is used by driver to check if hardware is compatible with the image. The header is only 16 bytes long and contains PCI Vendor ID, PCI Device ID and PCI Hardware Revision ID values to check. It also contains the CRC value of the raw image.
```

Comment:
Can you maybe link the header spec here? And explain what is the point of adding the header in a first place.